.github/workflows/build.yml

```diff
@@ -23,22 +23,6 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-  features:
```

Comment:
Why we are removing them?

Reply:
this removal is similar to it is like https://github.com/UBIDECO/rust-baid58/commit/0d9f2b2af90212212b0e9ca5723788bf07b5607b. atm we don't have any feature in this project.